### PR TITLE
feat: 🎸 Custom information field

### PR DESCRIPTION
**Pull request by dvonanderson**
✅ Closes: ICU-6881 ICU-6957

:tickets: [ICU-6881](https://hashicorp.atlassian.net/browse/ICU-6881), [ICU-6957](https://hashicorp.atlassian.net/browse/ICU-6957) 

## Description
Create custom information component for views that need read-only information listed.

:technologist: [Admin preview](https://www.figma.com/file/BB6W9sZwp75aOsXgCN8ecb/JSON-Credentials?node-id=1661%3A2780&t=yea0Yqe2Ah1p9PT9-0)

### Screenshots (if appropriate):

Screenshot 2022-11-30 at 12 26 27
Screenshot 2022-11-30 at 12 15 40
Screenshot 2022-11-30 at 12 26 20
Screenshot 2022-11-30 at 12 14 44
Screenshot 2022-11-30 at 12 14 19

## Previous look gave us a disabled radio card.
image

### New info-field component.
image

### If needed we can add a icon next to the label text
image

### CODE REVIEW

**Review by randallmorey — CHANGES_REQUESTED** [review]
I like how you integrated the icon into the component!  Only feedback is to tweak the layout/spacing when the icon is visible to align with the original design.  If possible, can we make the input color darker too?

**Comment by dvonanderson** [comment]
> I like how you integrated the icon into the component! Only feedback is to tweak the layout/spacing when the icon is visible to align with the original design. If possible, can we make the input color darker too?

Yes absolutely!

File: ui/admin/app/components/info-field/index.hbs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class='info-field'>
```
[the remark this answers is not in the document]
Comment (calcaide) [reply ui/admin/app/components/info-field/index.hbs:1]: Good catch, I miss that one

[the remark this answers is not in the document]
Comment (dvonanderson) [reply ui/admin/app/components/info-field/index.hbs:1]: yes Absolutely!

**Review by DhariniJeeva — APPROVED** [review]

**Review by cameronperera — APPROVED** [review]
LGTM

File: ui/admin/app/components/form/credential-store/static/index.hbs [hunk 2]
```diff
@@ -68,28 +68,22 @@
...
-          </form.radioGroup>
-        {{/if}}
-      </:body>
-    </form.fieldset>
+        </:body>
+      </form.fieldset>
+    {{else}}
+      <InfoField
+        @value={{t (concat 'resources.credential-store.types.' @model.type)}}
+        disabled={{form.disabled}}
```
Comment (cameronperera) [inline ui/admin/app/components/form/credential-store/static/index.hbs:76]: Should InfoField be disabled by default? Even when editing, we do not allow resources to change their type.

**Pull request merged**